ownCloud/Server List/ServerListTableViewController.swift

```diff
@@ -92,6 +92,8 @@ class ServerListTableViewController: UITableViewController, Themeable {
 		Theme.shared.register(client: self)
 
 		welcomeOverlayView.layoutSubviews()
+
+        self.tableView.reloadData()
```

Comment:
Please indent code.
Why is here a reload necessary now?

Reply:
@hosy fixed indentation (may be too aggresively). Reload is necessary, since usually you would have an LDAP display name after you have connected, and then you want to see this name when you return back to server list. If the table view won't be updated, you would see just a login name.